central_web_server/app.js

```diff
@@ -21,7 +21,7 @@ app.post('/api/:id', function(req, res){
 			return
 		}
 	});
-	console.log("Survey Recieved")
+	console.log("Survey Recieved: " + species_array.join(", "))
```

Comment:
It appears you're splitting and joining an array, would it be better to show all of the post data?